### ISSUE TITLE
Datascribe loading non-image media

### ISSUE THREAD

**Issue by mebrett** [chnm/Datascribe-module#25]
tl;dr datascribe is trying to display html media from the Omeka item - it doesn't work. 

I brought in items using the Tropy export to Omeka S, which sends along any notes you make as an HTML media item. I just looked at one of these in Datascribe, and it's treating the html media as a page to load in the dropdown/pagination on the _Add Record_ page. 
![Screen Shot 2020-04-14 at 10 45 17 AM](https://user-images.githubusercontent.com/2314070/79238273-2f4e8000-7e3d-11ea-8e11-a146cff9bf55.png)

If I page to the html media, I get a broken image symbol. 
![Screen Shot 2020-04-14 at 10 45 22 AM](https://user-images.githubusercontent.com/2314070/79238299-36758e00-7e3d-11ea-96ae-0f848cce6171.png)

 It seems to me there ought to be a way to stop it from trying to load non-image media in the first place?

**Comment by kimisgold** [comment]
I've added a whitelist of media types we want to display in the media viewer. @jimsafley and I discussed prioritizing the transcription of image media for at least the first release. The whitelist currently allows the following:

* 'image/bmp'
* 'image/gif'
* 'image/jpeg'
* 'image/png'
* 'image/svg+xml'

If there's more we'd like to include, we can discuss and close the issue after we come to a consensus.

**Comment by kimisgold** [comment]
for later releases:

* `image/tiff` and `image/pdf` 
  * ideal format for archival work
  * not supported by browsers inherently, we'd need to display the derivatives
     * we'd also need to split multi-page tiffs and pdfs into separate media attachments
  * @jimsafley has a module for managing multi-page attachments
